README.md

```diff
@@ -1567,6 +1567,7 @@ _Related: [Monitoring](#monitoring), [Bookmarks and Link Sharing](#bookmarks-and
 - [Hubleys](https://github.com/knrdl/hubleys-dashboard) - Self-hosted personal dashboards to organize links for multiple users via a central yaml config. `MIT` `Docker`
 - [Jump](https://github.com/daledavies/jump) - Yet another self-hosted startpage for your server designed to be simple, stylish, fast and secure. `MIT` `PHP`
 - [LinkStack](https://linkstack.org/) - Open-source, customizable, self-hosted alternative to services like Linktree and Manylink with an intuitive, easy to use user/admin interface. LinkStack allows you to link all your social media platforms easily accessible on one page. ([Demo](https://linksta.cc/), [Source Code](https://github.com/LinkStackOrg/LinkStack)) `AGPL-3.0` `PHP`
+- [LittleLink](https://github.com/sethcottle/littlelink/) - The original DIY Linktree alternative. A simplistic approach for links in bio with 100+ branded buttons. Host with GitHub Pages, self-host, one-click deploy, or use an awesome community fork with Docker. ([Demo](https://littlelink.io/), [Source Code](https://github.com/sethcottle/littlelink)) `MIT` `HTML/CSS`
```

Comment:
```suggestion
- [LittleLink](https://github.com/sethcottle/littlelink/) - DIY Linktree alternative. A simplistic approach for links in bio with 100+ branded buttons. ([Demo](https://littlelink.io/), [Source Code](https://github.com/sethcottle/littlelink)) `MIT` `HTML5/Docker`
```

More concise description, removed references to non-self-hosted options, updated language/platform tag to make it consistent with other entries.

Reply:
Thank you, this is great! I would just remove Docker from the tags. The core repo is just vanilla HTML/CSS, only community forks of LittleLink have out of the box Docker support.